### DISCUSSION TITLE
DeDRM not working

### DISCUSSION THREAD

**Discussion by pqrtrmn** [apprenticeharper/DeDRM_tools#2364] [General]
I'm definitely doing something wrong here but I can't get the plugin to work, any help appreciated!

[tmp8b3pz4x0.txt](https://github.com/user-attachments/files/17349605/tmp8b3pz4x0.txt)

**Comment by ElleKayEm** [comment]
That version is too old.  Use DeDRM 10.0.9: https://github.com/noDRM/DeDRM_tools/releases/tag/v10.0.9

**Reply by pqrtrmn** [reply]
Well that would be what I was doing wrong 🤦 

All working now - thanks!
